WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Posts/BlogDashboardCardFrameView.swift

```diff
@@ -0,0 +1,115 @@
+import UIKit
+import Gridicons
+
+/// A view that consists of the frame of a Dashboard card
+/// Title, icon and action can be customizable
+class BlogDashboardCardFrameView: UIView {
+
+    /// The main stack view, in which the header and the content
+    /// are appended to.
+    private lazy var mainStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        return stackView
+    }()
+
+    /// Header in which icon, title and chevron are added
+    private lazy var headerStackView: UIStackView = {
+        let topStackView = UIStackView()
+        topStackView.layoutMargins = Constants.headerPadding
+        topStackView.isLayoutMarginsRelativeArrangement = true
+        topStackView.spacing = Constants.headerHorizontalSpacing
+        topStackView.alignment = .center
+        return topStackView
+    }()
+
+    /// Card's icon image view
+    private lazy var iconImageView: UIImageView = {
+        let iconImageView = UIImageView(image: UIImage.gridicon(.posts, size: Constants.iconSize).withRenderingMode(.alwaysTemplate))
+        iconImageView.tintColor = .label
+        iconImageView.frame = CGRect(x: 0, y: 0, width: Constants.iconSize.width, height: Constants.iconSize.height)
+        iconImageView.setContentHuggingPriority(.defaultHigh, for: .horizontal)
+        iconImageView.isAccessibilityElement = false
+        return iconImageView
+    }()
+
+    /// Card's title
+    private lazy var titleLabel: UILabel = {
+        let titleLabel = UILabel()
+        titleLabel.text = "Card title"
```

Comment:
```suggestion
```

Reply:
Done in e4e6dd6a728634db9deffe3dce451693907965b7